src/app/elevation/qgselevationprofilewidget.cpp

```diff
@@ -237,6 +250,22 @@ QgsElevationProfileWidget::QgsElevationProfileWidget( const QString &name )
   topLayout->setSpacing( style()->pixelMetric( QStyle::PM_LayoutHorizontalSpacing ) );
   topLayout->addWidget( toolBar );
   topLayout->addStretch( 1 );
+
+  mInfoLabel = new QLabel();
+  topLayout->addWidget( mInfoLabel );
```

Comment:
Is this the best place to show display the result?
I think many users would benefit from also separately displaying `dZ` and `dXY` values.
Actually `dZ` is the most important value here, since other distances can me measured on the main canvas!

Reply:
> Is this the best place to show display the result?

It's not - but at this stage I'm so over budget for this project that I can't afford to tweak further

Reply:
Agreed with Stefanos that it is very desirable to have elevation and 2d distance reported also separately (e.g. if I need to find out building/tree height, without having to worry whether my measure line is perfectly vertical)

Reply:
Ok, i've gone for the same approach as the canvas measure tool uses now